label_studio/projects/models.py

```diff
@@ -375,7 +375,7 @@ def find_by_invite_url(cls, url):
 
     def reset_token(self):
         self.token = create_hash()
-        self.save()
+        self.save(update_fields=['token'])
```

Comment:
just a note that this is project.token (as opposed to organization.token), and it appears to be an unused field. Fine to leave as is :+1: